test/sales_analyst_test.rb

```diff
@@ -81,10 +86,11 @@ def test_average_item_price_standard_deviation
     sales_engine = SalesEngine.from_csv({
       :items     => "./test/fixtures/item_fixture.csv",
       :merchants => "./test/fixtures/merchant_fixture.csv",
+      :invoices  => "./test/fixtures/invoice_fixture.csv"
```

Comment:
Need to fix these syntactically

Reply:
Going to use the setup that I made on my tests and cut all that out. You'll see in the next merge.